lib/datasource/github-tags/index.ts

```diff
@@ -166,3 +157,34 @@ export async function getReleases({
   );
   return dependency;
 }
+
+function normalizeVersion(version: string): string {
+  return version?.replace(/^v/, '');
+}
+
+export async function getReleases(
+  config: GetReleasesConfig
+): Promise<ReleaseResult | null> {
+  const tagsResult = await getTags(config);
+
+  try {
+    // Fetch additional data from releases endpoint when possible
+    const releasesResult = await githubReleases.getReleases(config);
```

Comment:
Is it possible to call via the datasource index? e.g. to benefit from indexing. Not sure if we'll end up with a circular dependency though.

Reply:
Maybe it's possible, but then we need to make decisions about `registryUrls` strategy. And yes, it looks like something circular to me too.